ci/pipeline.yml

```diff
@@ -86,6 +86,18 @@ jobs:
             options:
               access_key_id: ((postgres-release-blobstore-user.username))
               secret_access_key: ((postgres-release-blobstore-user.password))
+    - task: bump-yq-package
+      file: postgres-release/ci/tasks/bump-yq-packages/task.yml
+      image: bosh-cli-registry-image
+      input_mapping:
+        yq-src: yq-4-src
+      params:
+        MAJOR_VERSION: 1
```

Comment:
If I'm following this correctly, I think the MAJOR_VERSION needs to be 4 rather than 1?

Reply:
Done